Unable to install with tarantoolctl rocks on Linux

```
$ tarantoolctl rocks install mqtt
<..cut..>
Install the project...
-- Install configuration: "RelWithDebInfo"
-- Installing: /home/alex/tmp/.rocks/share/tarantool/rocks/mqtt/scm-1/lib/mqtt/driver.so
-- Set runtime path of "/home/alex/tmp/.rocks/share/tarantool/rocks/mqtt/scm-1/lib/mqtt/driver.so" to ""
-- Installing: /home/alex/tmp/.rocks/share/tarantool/rocks/mqtt/scm-1/lua/mqtt/init.lua
CMake Error at third_party/mosquitto/build/cmake_install.cmake:41 (file):
  file cannot create directory: /usr/local/etc/mosquitto.  Maybe need
  administrative privileges.
Call Stack (most recent call first):
  mqtt/cmake_install.cmake:85 (include)
  cmake_install.cmake:42 (include)


gmake: *** [Makefile:118: install] Error 1

Error: Build error: Failed installing.
```

Got another problem before this one:

```
$ tarantoolctl rocks make
<...>
[ 41%] Linking C executable mosquitto_rr
/usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: CMakeFiles/mosquitto_rr.dir/pub_shared.c.o:/home/alex/p/tarantool-meta/mqtt/third_party/mosquitto/client/pub_shared.c:41: multiple definition of `cfg'; CMakeFiles/mosquitto_rr.dir/rr_client.c.o:/home/alex/p/tarantool-meta/mqtt/third_party/mosquitto/client/rr_client.c:50: first defined here
collect2: error: ld returned 1 exit status
```

The fix from https://github.com/eclipse/mosquitto/issues/1453 resolves the problem. We should update mosquitto to at least 1.6.8.

--- a/mqtt-scm-1.rockspec
+++ b/mqtt-scm-1.rockspec
@@ -18,7 +18,7 @@ build = {
         CMAKE_BUILD_TYPE="RelWithDebInfo";
         TARANTOOL_INSTALL_LIBDIR="$(LIBDIR)";
         TARANTOOL_INSTALL_LUADIR="$(LUADIR)";
-        STATIC_BUILD="$(STATIC_BUILD)";
+        STATIC_BUILD="ON";
     };
     platforms = {
         macosx = {

--- a/mqtt/CMakeLists.txt
+++ b/mqtt/CMakeLists.txt
@@ -4,6 +4,7 @@
 option(DOCUMENTATION "Build documentation?" OFF)
 
 option(WITH_STATIC_LIBRARIES "Build static versions of the libmosquitto/pp libraries?" OFF)
+option(WITH_PIC "Build the static library with PIC (Position Independent Code) enabled archives?" OFF)
 
 if( STATIC_BUILD )
 	set(WITH_STATIC_LIBRARIES ON)
@@ -20,7 +21,7 @@ add_library(driver SHARED driver.c)
 
 if( DEFINED STATIC_BUILD )
 	set(CMAKE_C_FLAGS "-ldl -lpthread")
-	add_subdirectory(../third_party/mosquitto ../third_party/mosquitto/build)
+	add_subdirectory(../third_party/mosquitto ../third_party/mosquitto/build EXCLUDE_FROM_ALL)
 	include_directories(../third_party/mosquitto/lib)
 	if( STATIC_BUILD )
 		target_link_libraries(driver libmosquitto_static ${LDFLAGS_EX})

